src/main/java/steps/TestResultsSteps.java

```diff
@@ -411,4 +419,108 @@ public void checkNextTestNumberIsValid(){
         String testNumber = response.jsonPath().getString("[0].testTypes[0].testNumber");
         assertThat(testNumber.equals(nextTestNumber)).isTrue();
     }
+
+    @Step
+    public void validateVehicleFieldValue(String key, String value) {
+        response.then().log().all();
```

Comment:
This won't show the bearer token in the log, right?

Reply:
nope, it's on response